src/api/app/models/bs_request.rb

```diff
@@ -32,6 +32,8 @@ class SaveError < APIException
 
   FINAL_REQUEST_STATES = %w(accepted declined superseded revoked)
 
+  VALID_REQUEST_STATES = [:new, :deleted, :declined, :accepted, :review, :revoked, :superseded]
```

Comment:
as I'm curious: why define one states as %w and one as symbol array?

Reply:
Probably because there was no coding style. And I think we still don't have one arrays.

I just copied this over;-)

Reply:
you mean @Ana06 has no cop for that? sad!

Reply:
We decide such things together;-)

There is a cop available. Either we didn't enable it yet, or it allows both ways by default.

Reply:
@coolo your wish is my command: https://github.com/openSUSE/open-build-service/pull/4004

Reply:
@bgeuken it was not enabled. But nice to have it. The `%w()` is the default one. Maybe you can change it here :pray:

Reply:
@Ana06 In principle yes, but I actually prefer the array syntax. So let's keep this discussion split from this changes.
It's two different things anyway:-)